client/src/main/proto/app/cash/backfila/service.proto

```diff
@@ -10,6 +10,7 @@ message Parameter {
   optional string name = 1;
   // Description to show in the UI.
   optional string description = 2;
+  optional bool required = 3;
```

Comment:
Not sure how to wire this in the client yet. It's just so we can indicate in the web UI when a param is required

Reply:
It can just be part of the Annotation?

Reply:
well in kotlin we know if it has a default or not, I just don't know where that is determined today. (won't be hard)